apps/nestjs-wallet/docs/tutorial.md

```diff
@@ -1,5 +1,56 @@
 # Tutorial
 
+## Business overview
+The business objective of this tutorial is to demonstrate how to issue a proof in the form of a Verifiable Credential (VC) to a known user connected to a portal that we will call "authority portal".
```

Comment:
I'm not sure what you mean by "a proof in the form of a Verifiable Credential". A Verifiable Credential contains a proof but isn't really a "form" of a proof, in my opinion.
Perhaps we can simplify it as shown below:
```suggestion
The business objective of this tutorial is to demonstrate how to issue a Verifiable Credential (VC) to a known user connected to a portal that we will call "authority portal".
```

Reply:
Ok, perfect